src/transaction/components/TransactionProvider.tsx

```diff
@@ -73,17 +84,48 @@ export function TransactionProvider({
     hash: writeContractTransactionHash || transactionHash,
   });
 
+  const getTransactionReceipts = useCallback(async () => {
+    const receipts = [];
+    for (const hash of transactionHashArray) {
+      try {
+        const txnReceipt = await waitForTransactionReceipt(config, {
```

Comment:
this generalizes for both wallet_getCallsStatus and eth_getTransactionReceipt right?

Reply:
this i'm not sure. i'm looking through the wagmi docs and haven't found any info yet.